README.md

```diff
@@ -1,135 +1,42 @@
-# Water Data for the Nation UI
+# Water Data For The Nation UI
 
 [![Build Status](https://travis-ci.org/usgs/waterdataui.svg?branch=master)](https://travis-ci.org/usgs/waterdataui)
-[![Coverage Status](https://coveralls.io/repos/github/usgs/waterdataui/badge.svg?branch=master)](https://coveralls.io/github/usgs/waterdataui?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/u`wdfn-server`sgs/waterdataui/badge.svg?branch=master)](https://coveralls.io/github/usgs/waterdataui?branch=master)
```

Comment:
I think this was changed by mistake.

Reply:
Is line 4 correct? It seems like a search/replace error.

Reply:
Thanks, didn't mean to do that!